core/arch/arm/plat-nuvoton/docs/README.md

```diff
@@ -0,0 +1,44 @@
+To build the OPTee OS for Nuvoton platform standalone (not as a part of the openbmc image).
```

Comment:
I believe this should go into the optee_doc git instead.

Reply:
I'm afraid this discussion is only going to delay the pull request.

Reply:
No problem, I moved it into another branch, it won't delay the pull request.